spotbugs/etc/findbugs.xml

```diff
@@ -864,9 +864,13 @@
           <BugPattern abbrev="IS" type="IS2_INCONSISTENT_SYNC" category="MT_CORRECTNESS"/>
           <BugPattern abbrev="NN" type="NN_NAKED_NOTIFY" category="MT_CORRECTNESS"/>
           <BugPattern abbrev="MS" type="MS_EXPOSE_REP" category="MALICIOUS_CODE"/>
+          <BugPattern abbrev="MS" type="MS_EXPOSE_BUF" category="MALICIOUS_CODE"/>
```

Comment:
Aren't abbreviations supposed to be unique?

Reply:
Not at all. There are lots of "MS" bugs from various detectors. Also "Dm", "Eq" etc. The more interesting question for me is that the original bugs from FindReturnRef are quite inconsistent: EI_EXPOSE_REP (abbr. EI) for returning a private instance level member. MS_EXPOSE_REP (abbr. MS) for doing the same with a static one. However: EI_EXPOSE_REP2 (abbr. EI2) for storing an external mutable at instance level but EI_EXPOSE_STATIC_REP (abbr. MS!!!) for doing the same in a static method. Also the bug descriptions are somewhat inconsistent: for the same kind of bug there are totally different descriptions for static and non-static versions. I tried to follow this way with the "buffer" error messages because I do not know the reason behind. However, I am not sure if we need separate messages for the buffer versions at all.